apps/swirl-docs/src/pages/api-docs/[apiSpec].tsx

```diff
@@ -73,17 +73,21 @@ async function generateSpecData(spec: string): Promise<ApiDocumentation> {
 
       const request = oasBuilder?.generateRequest(endpoint.operation);
       const responseExamples = endpoint.operation.getResponseExamples();
+      // const responseExamples: ResponseExamples = [];
 
       responseExamples.forEach((responseExample) => {
         const mediaTypes = Object.keys(responseExample.mediaTypes);
         const valueExample = responseExample.mediaTypes[
           mediaTypes[0]
         ] as Array<any>;
 
+        // TODO: fix the response examples on tasks
```

Comment:
**question:** Important?

Reply:
corresponds with the return lol thing^^ fixing right now